runtime.md

```diff
@@ -34,22 +34,20 @@ This is provided so that consumers can find the container's configuration and ro
 ```
 
 ## Lifecycle
-
-### Create
-
-Creates the container: file system, namespaces, cgroups, capabilities.
-
-### Start (process)
-
-Runs a process in a container.
-Can be invoked several times.
-
-### Stop (process)
-
-Not sure we need that from runc cli.
-Process is killed from the outside.
-
-This event needs to be captured by runc to run onstop event handlers.
+The lifecycle describes the timeline of events that happen from when a container is started to when it is stopped or crashes unexpectedly.
+
+* OCI compliant runtime is invoked by passing the bundle path as argument.
+* The container's runtime environment is created.
+* The prestart hooks are invoked in the host environment by the runtime.
```

Comment:
@vishh This is just a timeline of container lifecycle. For some runtimes like runc, this would be an internal implementation detail of what happens during start. I think we can tackle whether we want a separate create when we discuss the actions/API/interface.